Command/GenerateDoctrineEntityCommand.php

```diff
@@ -289,6 +310,88 @@ private function addFields(InputInterface $input, OutputInterface $output, Quest
                 $data['length'] = $questionHelper->ask($input, $output, $question);
             }
 
+            if ($type == 'one_to_many') {
+                $question = new Question($questionHelper->getQuestion('Target Entity class', ''), '');
+                $question->setAutocompleterValues($autocompleteEntities);
+                $data['targetEntity'] = $questionHelper->ask($input, $output, $question);
+
+                $question = new Question($questionHelper->getQuestion('Mapped by', ''), '');
+                $mappedBy = $questionHelper->ask($input, $output, $question);
+                if (empty($mappedBy) == false)
```

Comment:
You should do just a strict comparison of the user's input against the default value.